smart-contract-viem/hederaTestnet.js

```diff
@@ -0,0 +1,27 @@
+import 'dotenv/config'
+import { defineChain } from 'viem'
+
+const { RPC_URL } = process.env
+
+export const hederaTestnetChain = defineChain({
```

Comment:
This file may not be necessary, I'm previously added the definitions to viem itself: https://github.com/wevm/viem/blob/66e5f6ab/src/chains/index.ts#L211-L213

(note that you can see how this is used in the `connect-viem` code snippet: https://github.com/hedera-dev/hedera-code-snippets/blob/ab0ec69b/connect-viem/connect.js#L2 )